### PR TITLE
Contacts and Suggested Friends

### PR DESCRIPTION
**Pull request by JakasaurusRex**
## Description
You can now get suggestions for friends on your add friends tab. The suggested friends will be mutual friends that you have or contacts that have created Piccy accounts. To do this I implemented the ContactKit pod. All of the new features include:
- [X] Suggested friends list
- [X] Add contacts that have created Piccy accounts
- [X] Add mutual friends
- [X] Updated UI in Piccy and PFP selection page so that users who skip the alert will have an idea of what to do. 

## Milestones
I am now complete with the friends page and all features associated with users I believe.

## Test Plan
I had to fix an edge case where users who didnt have mutual friends but had contacts would break the app but that is fixed. Also if you have yourself as a contact I accounted for that. 

Screen Shot 2022-07-27 at 5 42 21 PM

### CODE REVIEW

File: Piccy/Piccy/View Controllers/Friends Page/FriendsViewController.m [hunk 1]
```diff
@@ -291,7 +332,7 @@ - (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger
     if(self.segCtrl.selectedSegmentIndex == 1 || self.segCtrl.selectedSegmentIndex == 2) {
         return [self.friends count];
     }
-    if(section == 0) {
+    if(section == 1) {
```
Comment (sipingggg) [inline Piccy/Piccy/View Controllers/Friends Page/FriendsViewController.m:355]: what does 1 mean? change it in your enum diff later

**Review by sipingggg — APPROVED** [review]
